### PR TITLE
Enhanced oracle/unique-key-conflicts-test for fall forward scenario

### PR DESCRIPTION
**Pull request by sanyamsinghal**
- added more events to source_delta.sql, not just conflicting ones, to mimic real world scenarios
- added validate.py script after snapshot import is done

Jenkins change - https://github.com/yugabyte/jenkins-helpers/pull/769

### CODE REVIEW

**Comment by sanyamsinghal** [comment]
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline-test/905/

File: migtests/lib/oracle.py [hunk 1]
```diff
@@ -90,6 +90,11 @@ def row_count_of_all_tables(self, schema_name) -> Dict[str, int]:
         tables = ['"{}"'.format(table[0]) for table in cur.fetchall()]
         #return tables in lower case to match EXPECTED values Todo handled it properly later
         return {table[1:-1].lower(): self.get_row_count(table, schema_name) for table in tables}
+    
+    def get_distinct_values__of_column_of_table(self, table_name, column_name, schema_name) -> List[Any]:
```
Comment (shubham-yb) [inline migtests/lib/oracle.py:94]: nit: `get_distinct_values_of_column_of_table`

**Comment by shubham-yb** [comment]
Looks good overall. Thanks for adding these!
Please consider adding these cases as well

2 different unique constraints on the same table
2 different unique indexes on the same table

**Comment by sanyamsinghal** [comment]
> 2 different unique constraints on the same table
2 different unique indexes on the same table

@shubham-yb  Add these will be a significant amount of work, since need add to from source till fall forward.

Will add a TODO for these for now.



Moreover for these type of cases we can have unit tests also. Basically testing the internal functions in code.l

**Comment by sanyamsinghal** [comment]
latest: https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline-test/942/

**Review by shubham-yb — APPROVED** [review]
LGTM! Just run once with FallB enabled, I don't think much change will be required. If there's significant change we can take it up later.

**Pull request merged**